tools/manage_clvm.py

```diff
@@ -164,4 +181,52 @@ def main() -> int:
     return 1 if overall_fail else 0
 
 
+@main.command()
+def build() -> int:
```

Comment:
is this function triggered by CI? i.e. is it demonstrated to work and produce identical output?

Reply:
Nope.  This was a side step that wasn't removed.